### PR TITLE
fix(api): Allow superusers to access org events endpoint

### PR DESCRIPTION
**Pull request by macqueen**
fixes SENTRY-7X7

### CODE REVIEW

**Comment by MeredithAnya** [comment]
I think you can do `self.login_as(user=self.user, superuser=True)`

**Comment by macqueen** [comment]
oh cool thanks @MeredithAnya! updated

File: src/sentry/api/endpoints/organization_events.py [hunk 1]
```diff
@@ -22,10 +22,13 @@ def get_project_ids(self, request, organization):
...
 
-        om_role = OrganizationMember.objects.filter(
-            user=request.user,
-            organization=organization,
-        ).values_list('role', flat=True).get()
+        try:
+            om_role = OrganizationMember.objects.filter(
+                user=request.user,
+                organization=organization,
+            ).values_list('role', flat=True).get()
```
Comment (dcramer) [inline src/sentry/api/endpoints/organization_events.py]: does .first() work on this kind of queryset? if so it lets you simplify this code mostly to what it was before

**Review by dcramer — APPROVED** [review]

File: src/sentry/api/endpoints/organization_events.py (hunk 1 above)
Comment (mattrobenolt) [reply src/sentry/api/endpoints/organization_events.py]: Please no, `.first()` adds an implicit `ORDER BY id` which changes the query plans sometimes into something bad and unintended.

Comment (dcramer) [reply src/sentry/api/endpoints/organization_events.py]: We can just override the manager but using first reduces a lot of boilerplate code.

**Pull request merged**